samples/httpd/main.c

```diff
@@ -67,6 +68,8 @@ void main(void)
 	sys_sem_wait(&init_complete);
 	sys_sem_free(&init_complete);
 
+        XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
```

Comment:
nit: Mix of tab + spaces here. Seems to be the only sample using tabs, but we should probably be consistent for the file and create a PR later to change the styling of this file down the road if changing this is important enough.